upgrade loris database with release_patch (#5883)
aces/Loris: README.md

7. Follow the [Setup Guide in the LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) to complete your post-installation setup and configuration, and for more documentation.

> ### Upgrade Step

Reviewer: I think this should be a one line mention of this step/process/script -- and point to the rest of this content under the markdown docs/ (what Rida calls the "new wiki)

> Backing Up the Database Before Upgrading the Database Schemas.

Reviewer 2: ```suggestion
It is recommended to do a backup of the Database before upgrading the database schemas.
```

> If you have installed previous version Loris, you can upgrade Loris to the latest in the following ways:

Reviewer 2: ```suggestion
If you have installed a previous version Loris, you can upgrade Loris to the latest in the following ways:
```

> If you have installed a previous version Loris, you can upgrade Loris to the latest in the following ways:

Reviewer 2: ```suggestion
If you have installed a previous version of LORIS, you can upgrade LORIS to the latest in the following ways:
```

> 1. Using Git command checkout to the latest version of Loris.

Reviewer 2: ```suggestion
1. Using Git command checkout the latest version of Loris.
```

 ```bash

>  git checkout aces/22.0-release

Reviewer: this is on the master branch.  this number in this readme will need to be updated right before every release.  Can we find an easier way to write/maintain this?

Reviewer 3: There's also no guarantee the remote will be named `aces`.

 ```

> 2.-- Manual upgrade(optional): Run upgrade.sql files in your database in the /var/www/$projectname/SQL/Release_patches/ directory.

Reviewer 2: ```suggestion
2.-- Manual upgrade(optional): Run upgrade.sql files in `/var/www/$projectname/SQL/Release_patches/` directory in your database.
```

>   -- Automatic upgrade(optional): Run Ex.(php loris_db_upgrade.php 21.0) in /var/www/$projectname/tools/ directory.

Reviewer 2: ```suggestion
  -- Automatic upgrade(optional): Run the upgrade script in `/var/www/$projectname/tools/` directory. Ex. `php loris_db_upgrade.php 21.0`
```

 ```bash
 cd /var/www/$projectname/tools/

>  php loris_db_upgrade.php 

Reviewer: which database permissions are required to run the script?  are those of the `config.xml` credential (`lorisdbuser`) sufficient?

Author: I will add it later.

Reviewer 3: The script should use `adminUser` and `adminPassword` ([formerly known as](#5785) "quatUser").
